tasks/configuration.yml

```diff
@@ -30,6 +30,7 @@
     dest: "{{ nginx_conf_dir }}/sites-enabled/{{ item.key }}.conf"
   with_dict: "{{ nginx_sites }}"
   when: (item.key not in nginx_remove_sites) and (item.key not in nginx_disabled_sites)
+  ignore_errors: "{{ ansible_check_mode }}"
```

Comment:
Why ?

Reply:
If you're running Ansible in "check mode" (`ansible-playbook main.yml -i inventory/hosts -b --check`) you're executing the task "Create the configurations for sites" which works fine; but the task "Create links for sites-enabled" is failing since the source files are not available.

`
...
"msg": "src file does not exist, use \"force=yes\" if you really want to create the link: /etc/nginx/sites-available/XXX.conf",
"path": "/etc/nginx/sites-enabled/XXX.conf",
"src": "/etc/nginx/sites-available/XXX.conf",
"state": "absent"
...`

Reply:
Good point !